src/listener.rs

```diff
@@ -410,7 +422,23 @@ impl Listener {
                                 feature: "UNIX-like platform",
                             }
                         }
-                    }
+                    },
+                    ListenerAddress::Vsock{..} => {
+                        #[cfg(any(target_os = "linux", target_os = "android", target_os = "macos"))]
+                        {
+                            MissingCompileTimeFeature {
+                                reason: "use vsock socket",
+                                feature: "vsock",
+                            }
+                        }
+                        #[cfg(not(any(target_os = "linux", target_os = "android", target_os = "macos")))]
+                        {
+                            MissingPlatformSupport {
+                                reason: "use vsock socket",
+                                feature: "linux platform",
```

Comment:
May be misleading to list only "linux" in the error message when it actually supports more platforms.

Reply:
Would change for "linux, android or darwin" platform.